gender_novels/corpora/sample_novels/sample_novels.csv

```diff
@@ -1,100 +1,96 @@
 author,date,title,country_publication,author_gender,filename,notes
 "Alcott, Louisa May",1868,Little Women,United States,female,alcott_littlewomen.txt,
-"Alcott, Louisa May",1886,"Jo's Boys, and How They Turned Out",United States,female,alcott_josboys.txt
-"Alcott, Louisa May",1871,"Little Men: Life at Plumfield with Jo's Boys",United States,female,alcott_littlemen.txt
+,,,,,,

```

Comment:
Please delete the two empty lines--otherwise, the Corpus object will think there are no titles or authors.